contractcourt/chain_arbitrator.go

```diff
@@ -641,6 +641,85 @@ func (c *ChainArbitrator) Start() error {
 	return nil
 }
 
+// dispatchBlocks consumes a block epoch notification stream and dispatches
+// blocks to each of the chain arb's active channel arbitrators. This function
+// must be run in a goroutine.
+func (c *ChainArbitrator) dispatchBlocks(
+	blockEpoch *chainntnfs.BlockEpochEvent) {
+
+	// On exit, cancel our blocks subscription and shutdown all of our
+	// block queues so that the arbitrators know they will no longer be
```

Comment:
Will no longer receive? They should continue to consume updates backed by only a _single_ chain notifier epoch impl.

Reply:
Not sure if I'm following here? 

If `dispatchBlocks` exits, our block epoch stream has been cancelled, or the chain arb itself has received the shutdown signal. Most likely we are shutting down, so we want to let all the consuming goroutines (ie, `channelAttendant`) that we are exiting so that it can return? As is, we already [do this](https://github.com/carlaKC/lnd/blob/master/contractcourt/channel_arbitrator.go#L2124) if our epoch subscription is shutdown.